src/compiler/crystal/semantic/call_error.cr

```diff
@@ -38,7 +38,7 @@ end
 
 class Crystal::Call
   def raise_matches_not_found(owner, def_name, arg_types, named_args_types, matches = nil, with_literals = false)
-    # Special case: Foo+:Class#new
+    # Special case: Foo+.class#new
```

Comment:
This is not exactly source code syntax, but it looks as if...

Reply:
The `+` syntax is a different issue